examples/getting_started/pendulum.py

```diff
@@ -68,17 +68,24 @@ def prepare_ocp(biorbd_model_path, final_time, number_shooting_points, nb_thread
 
     # --- Solve the program --- #
     tic = time()
-    sol = ocp.solve(show_online_optim=False)
+    sol, iterations = ocp.solve(show_online_optim=True, save_iterations=True)
     toc = time() - tic
     print(f"Time to solve : {toc}sec")
```

Comment:
Can you test the iteration time when the optimization goes to 1000 iterations (compare to "save_iterations=False")